src/covid_model_seiir_pipeline/regression/task.py

```diff
@@ -0,0 +1,116 @@
+from argparse import ArgumentParser, Namespace
```

Comment:
was this just a rename of `beta_regression.py`?

Reply:
Sorry, this was super sloppy of me.

I axed the diagnostics task, at which point we didn't need a subpackage for tasks.  This is substantially rewritten from the original though.